# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mathieu/streamtuner/ignore-void-pointer.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/plugins/local/local.c.orig	2004-12-18 06:28:45.956675320 +0100
+++ ./src/plugins/local/local.c	2004-12-18 06:29:02.772118984 +0100
@@ -838,9 +838,7 @@
 	    }
 	}
 
-      status = taglib_file_save(file);
-      if (! status)
-	g_set_error(err, 0, 0, _("unable to save file"));
+      taglib_file_save(file);
     }
   else
     {
